lib/engine/game/g_1893/share_pool.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+require_relative 'share_pool'
+
+module Engine
+  module Game
+    module G1893
+      class SharePool < Engine::SharePool
```

Comment:
again the problem is adsk, so delete this class as well

Reply:
As mentioned above, making AdsK company created other problems.

But yes, it has been deleted.